ttnn/cpp/ttnn/operations/conv/conv2d/device/conv2d_op_sharded_program_factory.cpp

```diff
@@ -935,8 +935,7 @@ operation::ProgramWithCallbacks multi_core_optimized_conv_sharded_v2_impl(
 
     if (fully_buffer_weights) {
         num_weight_cb_tiles *= window_outer;
-    } else if (per_core_out_matrix_width_ntiles < 5 && per_core_out_matrix_height_ntiles < 22) {  // Q: where are these
-                                                                                                  // numbers from?
+    } else if (enable_act_double_buffer) {
```

Comment:
Do you mean `enable_weights_double_buffer` ?? If not, I am not seeing that variable used anywhere else.

Reply:
yep, thx!